src/kw_config_loader.sh

```diff
@@ -127,7 +127,16 @@ function load_configuration()
 {
   parse_configuration "$KW_ETC_DIR/$CONFIG_FILENAME"
   parse_configuration "$HOME/.kw/$CONFIG_FILENAME"
-  parse_configuration "$PWD/$CONFIG_FILENAME"
+
+  # New users may not have kworkflow.config at $PWD
```

Comment:
It's something like: new users may not have the file, so we check before running the command that relies on the file so it doesn't throw an error for not having found that file.
The second comment `# Old users may not have used kw init yet, so they wouldn't have .kw` follows the same logic: since old users may not have .kw, we'll check if the directory exists before running a command that uses it. Does that make sense?